### Matthew Dalquist (mdalquist@ucsd.edu)
### Section A06 - Utkrisht Rajkumar and Thiago Mosquiero
**1. What is the most interesting topic covered in your domain this quarter?**
The topic I have found most interesting this quarter was the methods used to represent DNA to be used in machine learning. While there are  more than one method, we studied the representation of DNA using graphs. The genome can be split into regions of 10,000 base pairs, and each region may contain a gene or set of genes. These regions make up the nodes of the graph, and the edges represent known gene interactions between regions of DNA. The paper we read used a GCN on this graph to predict long range genetic interactions.

**2. Describe a potential investigation you would like to pursue for your Quarter 2 Project.**
A potential idea for our quarter 2 project is applying transformers to GNNs in order to classify which DNA strings are extrachromosomal DNA (ecDNA). ecDNA is known to have some link to cancer, but at this point, the problem of identifying ecDNA in the genome has been difficult.

**3. What is a potential change youâ€™d make to the approach taken in your current Quarter 1 Project?**
Our current idea for the change we would make to the method we studied (GC MERGE) would be to add an attention layer to the network. This would allow us to capture more information about interactions between nodes other than if there is simply a known genetic interaction between 2 genomic regions.

**4. What other techniques would you be interested in using in your project?**
I am interested in doing data exploration for the project. I don't know much about the dataset we use, or much about how DNA datasets are structured. It will be fun to explore it and run statistical tests to see what we can find out.
